neqo-crypto/src/lib.rs

```diff
@@ -59,7 +59,7 @@ pub use self::{
     ssl::Opt,
 };
 
-const MINIMUM_NSS_VERSION: &str = "3.97";
+include!("../min_version.rs");
```

Comment:
Again, I think that it is better to use:

```rust
#[path = "../min_version.rs"]
mod min_version;
use min_version::MINIMUM_NSS_VERSION;
```

Reply:
Do you have a preference in which directory `min_version.txt` and `min_version.rs` live? I wasn't sure whether to put one or both under `src`.